scripts/docs/build-sources.mjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/* eslint-env node */
+
+import upath from 'upath';
+import fs from 'fs-extra';
+import { CKEDITOR5_ROOT_PATH, CKEDITOR5_COMMERCIAL_PATH } from '../constants.mjs';
+import generateCKEditor5DocsBuild from './generate-ckeditor5-docs-build.mjs';
+
+export default async function buildSources() {
+	const { version } = await fs.readJson( upath.join( CKEDITOR5_ROOT_PATH, 'package.json' ) );
+	const basePath = upath.join( CKEDITOR5_ROOT_PATH, 'build/docs/ckeditor5', version, 'assets' );
+
+	const output = getOutputFactory( basePath );
+
+	await generateCKEditor5DocsBuild( output( 'ckeditor5/ckeditor5.js' ) );
+
+	if ( await fs.pathExists( CKEDITOR5_COMMERCIAL_PATH ) ) {
+		const { default: generateCKEditor5PremiumFeaturesDocsBuild } = await import(
+			'../../external/ckeditor5-commercial/scripts/docs/generate-ckeditor5-premium-features-docs-build.mjs'
```

Comment:
I'd be better to build this path from the constants we already have. I haven't tested it, so please check if it works and make updates if necessary.

Remember to add `import { pathToFileURL } from 'url';`

```suggestion
			pathToFileURL( upath.join( CKEDITOR5_COMMERCIAL_PATH, 'scripts', 'docs', 'generate-ckeditor5-premium-features-docs-build.mjs' ) );
```